Guardian/API/Models.swift

```diff
@@ -23,7 +23,7 @@
 import Foundation
 
 struct PushCredentials: Codable {
-    let service = "APNS"
+    private(set) var service = "APNS"
```

Comment:
Is this being modified somewhere?

Reply:
It produces warning because of conforming to Codable. It is modified in the process of state restoring.